src/structure_managers/property_base.hh

```diff
@@ -91,6 +92,12 @@ namespace rascal {
       this->update_nb_comp();
     }
 
+    inline void set_shape(const Dim_t & nb_row, const Dim_t & nb_col) {
```

Comment:
I guess this should serve as a shorthand instead of using `set_nb_row()` and `set_nb_col()`?
Makes sense.